app/tests/Core/documentTests.js

```diff
@@ -1647,6 +1648,14 @@ export default [
           product_status: {
             fixed: ['CSAFPID-0001'],
           },
+          ...(doc === minimalVexDoc
+            ? {
+                id: {
+                  system_name: 'My system',
+                  text: 'My system test',
```

Comment:
Just to make the usage clear => link to example from spec.
```suggestion
                  system_name: 'GitHub Issue',
                  text: 'oasis-tcs/csaf#210',